app/controllers/review_chats_controller.rb

```diff
@@ -0,0 +1,57 @@
+class ReviewChatsController < ApplicationController
+  helper :review_chats
+  before_action :set_review_chat, only: [:show, :edit, :update, :destroy]
+
+  # GET /review_chats
+  def action_allowed?
+    review_chat = ReviewChat.find(params[:id])
+    allowed_users=Array.new
+    team_id=review_chat.team_id
+    teams_users = TeamsUser.where(team_id: team_id)
+    teams_users.each do |teams_user|
+      allowed_users << User.find(teams_user.user_id).id
+    end
+    allowed_users << Participant.find(review_chat.reviewer_id).user_id
+    current_role_name.eql? 'Student' and allowed_users.include?(session[:user].id)
+  end
+
+  def index
+    @review_chats = ReviewChat.all
+  end
+
+  # GET /review_chats/1
+  def show
+    @review_chat = ReviewChat.find(params[:id])
+    @assignment_id=@review_chat.assignment_id
+    @reviewer_id=@review_chat.reviewer_id
+    @team_id=@review_chat.team_id
+    @chat_log=ReviewChat.where(:reviewer_id => @reviewer_id).where(:team_id => @team_id)
```

Comment:
We created a method get_chat_log in the model and have already implemented this and pushed to git